MomsSpaghetti/Pages/Index.cshtml

```diff
@@ -225,15 +225,31 @@
 
                                 @foreach (var ingredient in Model.Recipe.Ingredients)
                                 {
+                                    @*<li class="recipe__item">
+                    <svg class="recipe__icon">
+                        <use href="img/icons.svg#icon-check"></use>
+                    </svg>
```

Comment:
What's svg?

Reply:
It has not been added by me. It was already there in the code.